packages/components/src/renderer/index.tsx

```diff
@@ -48,7 +49,11 @@ export class Renderer {
 
   public register(handlers: Partial<{ [key in MessageType]: MessageTypeHandler<key> }>) {
     for (const type in handlers) {
-      this.set(type as MessageType, handlers[type])
+      this.set(
+        type as MessageType,
+        // TODO: fix typings here. This produces an implicit any
+        (handlers as any)[type]
```

Comment:
There's voodoo typings here that I couldn't make work in 5 minutes

Reply:
fixed